index.js

```diff
@@ -107,7 +107,18 @@ module.exports = function (fastify, opts, next) {
         }
 
         try {
-          const plugin = require(file)
+          const content = require(file)
+          let plugin
+
+          if (content && typeof content === 'object' && content.method) {
```

Comment:
This check for object-ness is not suffcient. An array will pass this check. You should `Object.prototoype.toString.apply(content) === true`.

Reply:
What do you think of:

```
Object.prototype.toString.apply(content) === '[object Object]' &&
Object.prototype.hasOwnProperty.call(content, 'method')
```

?

Reply:
Yes, that's what I meant 🤦‍♂. It's still early morning for me over here.

Reply:
No worries :) thanks for the hint. I thought about that before but then though this might be already sufficient but you're right. Adding both checks now.